src/octoprint/plugins/tracking/__init__.py

```diff
@@ -473,6 +475,13 @@ def _track_printer_event(self, event, payload):
             if self._printer_connection_parameters:
                 args["printer_port"] = self._printer_connection_parameters["port"]
                 args["printer_baudrate"] = self._printer_connection_parameters["baudrate"]
+
+            profile = self._printer_profile_manager.get_current().copy()
+            for k in ["id", "name", "model"]:
+                # Scrub these out since they could be identifiable information
+                profile.pop(k)
+            args["printer_profile"] = profile
```

Comment:
That will need a further deep dive to be flattened. We cannot push hierarchical data like this over to the tracking subsystem, it will end up as one `repr` string, completely unsearchable or otherwise usable. 

I suggest we concentrate on some key infos like extruder count, bed size, presence of heated bed and heated chamber?